build.sbt

```diff
@@ -1,23 +1,62 @@
 import xerial.sbt.Sonatype._
+import Dependencies._
 
-cancelable in Global := true
-onLoad in Global ~= (_ andThen ("project core" :: _))
```

Comment:
This was preventing `scalafmtSbtCheck` from having visibility into the top-level `*.sbt` and `project/*.scala` files.

See the alternative approach below with a root project.